lib/pbench/server/api/resources/alembic.ini

```diff
@@ -0,0 +1,85 @@
+# A generic, single database configuration.
```

Comment:
I don't quite understand what we're "migrating" here?

Reply:
We are not migrating anything right now but if we make any changes to db definitions, then we would have to do the migrations first otherwise the queries would break. SO i just created a framework for it. Btw the file is auto-generated with `alembic init alembic` command we just have to tweak the required parameters. We can run the migrations like this `alembic revision — autogenerate -m “Add new column”` and it will make the migration file in the revision directory. We can go back and forth between the revisions